homeassistant/components/renault/renault_coordinator.py

```diff
@@ -0,0 +1,66 @@
+"""Proxy to handle account communication with Renault servers."""
+from __future__ import annotations
+
+from collections.abc import Awaitable
+from datetime import timedelta
+import logging
+from typing import Callable
+
+from renault_api.kamereon.exceptions import (
+    AccessDeniedException,
+    KamereonResponseException,
+    NotSupportedException,
+)
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import (
+    DataUpdateCoordinator,
+    T,
```

Comment:
I'm not sure if importing such unconstrained `TypeVar` is a good idea. Maybe define a new one instead?

Reply:
Should be good in cd74538ab371ebbf09b211e411b8f5d7a29711e4

Reply:
Yes, thanks!